mtl_facilitate_workgroup/facilitator_say/mtl_session02_say.md

```diff
@@ -34,9 +34,7 @@ Hello! I'm __________ and I'm __________. Navigating to the Learner See Guide fo
 ### After this session, you will be able to:
 
 1. Describe the decisions your team made in producing your team data table.
-
 2. Test out whether your expectations about team historical trends are displayed in the "viz" tabs.
```

Comment:
@branscombj I changed this back. Rita made sure all of the learning objectives match across SEE/SAY/Master Brochures and sent them to Elizabeth.